cspatients/views.py

```diff
@@ -59,17 +59,19 @@ def view(request):
 @login_required(login_url="/cspatients/login")
 def patient(request, patient_id):
     template = loader.get_template("cspatients/patient.html")
+    status_code = 200
     try:
         patiententry = PatientEntry.objects.get(patient_id=patient_id)
-        context = {
-            "patiententry": patiententry,
-        }
-        status_code = 200
-    except Patient.DoesNotExist:
-        context = {
-            "patiententry": False,
-        }
-        status_code = 400
+    except PatientEntry.MultipleObjectsReturned:
```

Comment:
I just want to check the situation we are handling here - if a patient is entered into the system twice (or comes to the hospital more than once?), it will return the most recently changed entry?

Reply:
So suppose a patient gets an operation, but after that is booked for another one. Rowan explained a situation where a patient can get an operation done, but maybe because of a complication has to be booked for another. This means that for that same patient there will be another entry for the other operation. Now if there are multiple entries for the same patient, using get on the queries won't work. I am now considering maybe it is better to ways assume it returns multiple entries and pick the latest one?

Reply:
Yeah, so I think that instead of catching an exception, you could simply run something like the following:
`patiententry = PatientEntry.objects.filter(patient_id=patient_id).order_by('-decision_time').first()`